### PR TITLE
Add some location wisdom

### PR DESCRIPTION
**Pull request by robertkowalski**
:)

### CODE REVIEW

**Comment by robertkowalski** [comment]
its 3 now @klyrr

File: location.md [hunk 1]
```diff
@@ -14,6 +14,28 @@
...
 ### Selection
 
-Have a selection of dates available that you can use for choosing locations.
+Have a selection of dates available that you can use for choosing
+locations.
 
 ## Locations
+
+Depending on the conference setup and type your needs for a location
+will change.
```
Comment (klyrr) [inline location.md]: you have some weird line breaks here and in the paragraph below. Can you remove them?

Comment (robertkowalski) [reply location.md]: its 72 chars by line, its quite common, i don't think thats "weird", but I can switch to no line breaks at all if you want, to have it in line with the other texts

Comment (klyrr) [inline location.md]: Yes, please :)

**Pull request merged**